### PR TITLE
VZ-9602: BFS: Create new Ingress NGINX controller v1.7.1

### PR DESCRIPTION
**Pull request by saidu941**
This PR contains changes to uptake new version of controller from 1.3.1 to 1.7.1

### CODE REVIEW

File: platform-operator/thirdparty/charts/ingress-nginx/ci/controller-admission-tls-cert-manager-values.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+controller:
+  admissionWebhooks:
+    certManager:
+      enabled: true
```
Comment (michael-cico) [inline platform-operator/thirdparty/charts/ingress-nginx/ci/controller-admission-tls-cert-manager-values.yaml:4]: We might need to see if this needs to allow overrides for the case where `spec.components.clusterIssuer.enabled=false` in the Verrazzano CR.

Cert-Manager can be disabled, so we might need to be able to toggle this in the append overrides.  Be sure to run the a-la-carte pipeline

File: platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+controller:
+  image:
+    repository: ingress-controller/controller
+    tag: 1.0.0-dev
+    digest: null
+  service:
+    type: ClusterIP
+  containerSecurityContext:
```
Comment (michael-cico) [inline platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml:8]: We also might need to see if we need to tweak our security context settings in the templates.

**Review by mbenmclean — APPROVED** [review]

File: platform-operator/thirdparty/charts/ingress-nginx/ci/controller-admission-tls-cert-manager-values.yaml (hunk 1 above)
Comment (saidu941) [reply platform-operator/thirdparty/charts/ingress-nginx/ci/controller-admission-tls-cert-manager-values.yaml]: Updated pr: https://github.com/verrazzano/verrazzano/pull/6079/commits/cfbe09ff84b0196e3b53ee2413939a08e040d009

File: platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml (hunk 2 above)
Comment (saidu941) [reply platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml:8]: Sorry, I couldn't understand your suggestion. Could you please elaborate?

Comment (saidu941) [reply platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml:8]: I'm not sure I understood this, could you please elaborate

Comment (saidu941) [reply platform-operator/thirdparty/charts/ingress-nginx/ci/deployment-extra-modules-default-container-sec-context.yaml:8]: @pfmackin could you please verify the security settings for NGINX looks good?

**Review by michael-cico — DISMISSED** [review]

**Review by michael-cico — DISMISSED** [review]

**Review by fredtibbitts — DISMISSED** [review]

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**